state_machine/applications/example/Tasks/transition_task.py

```diff
@@ -1,14 +1,15 @@
 from lib.template_task import Task
+import lib.pycubed as cubesat
 
 
 class task(Task):
     name = 'transition'
     color = 'blue'
 
     async def main_task(self):
-        if self.cubesat.state_machine.state == 'Normal':
+        if cubesat.state_machine.state == 'Normal':
```

Comment:
I don't think I saw cubesat.state_machine defined anywhere in the driver file (+ I know that I haven't defined it). It looks like this was originally an attribute of the cubesat object that was created after the Satellite object was initialized, but I'm not sure it's properly initialized anywhere now that we've hidden away the class definition. Please let me know if there's anything additional you need me to define to accomodate this.